hubtools/src/caboose.rs

```diff
@@ -44,6 +44,14 @@ impl Caboose {
         self.get_tag(tags::VERS)
     }
 
+    pub fn sign(&self) -> Result<&[u8], CabooseError> {
+        self.get_tag(tags::SIGN)
+    }
```

Comment:
I think the base is still incorrect, this was addedin https://github.com/oxidecomputer/hubtools/commit/54296a058594aff7131de3931a97cc7bf2f42ae6

Reply:
This PR is at fc057a0 now. I had cherry-picked the `SIGN` PR. I've since rebased onto main.